readthedocs/projects/migrations/0024_alter_domain_model.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
Name suggestion: `0024_alter_and_project_models.py`

Reply:
`...alter_and` domain and project, right?